src/PresentationalComponents/Notification/Notification.js

```diff
@@ -109,20 +113,25 @@ const Notification = () => {
   return (
     <React.Fragment>
       <PageHeader>
-        <PageHeaderTitle title="Notification Insights" />
+        <Split>
+          <SplitItem isFilled>
+            <PageHeaderTitle
+              title={`My Notifications | ${bundleDisplayTitle.title}`}
+            />
+            <StackItem>
+              This service allows you to opt-in and out of recieving
+              notifications. Your Organization Administrator has configured
+              which notifications you can or can not recieve in their
+              <a href={`/settings/notifications/${bundleName}`}> Settings</a>.
```

Comment:
We need to take into account the beta/stable status here. Not sure how it's done here, but you can search for `beta` and see how is handled.